### PR TITLE
Migrating Repo

### PR DESCRIPTION
**Pull request by jeradrutnam**

### CODE REVIEW

File: apps/user-portal/.jshintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "esversion": 6
+}
```
Comment (thanujalk) [inline apps/user-portal/.jshintrc]: add a new line at the EOF

File: apps/user-portal/server.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+'use strict';
```
Comment (thanujalk) [inline apps/user-portal/server.js]: do we need to add license header here?

File: apps/user-portal/src/app.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// import { DefaultTheme, Logo, Theme, } from "@wso2is/theme";
```
Comment (thanujalk) [inline apps/user-portal/src/app.tsx]: Remove the commented line.

File: apps/user-portal/src/locales/en.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+    "errors": {
+        "login": {
+            "emptyPassword": "Password cannot be empty",
+            "emptyUsername": "Username cannot be empty",
+            "invalidCredentials": "Invalid username or password, please type again",
+            "loginFailed": "We're sorry we couldn't get you logged in"
+        }
+    }
+}
```
Comment (thanujalk) [inline apps/user-portal/src/locales/en.json]: Add a new line at the EOF.

**Comment by thanujalk** [comment]
Better to use code formatting for the js files.

File: apps/user-portal/.jshintrc (hunk 1 above)
Comment (jeradrutnam) [reply apps/user-portal/.jshintrc]: Fixed

File: apps/user-portal/server.js (hunk 2 above)
Comment (jeradrutnam) [reply apps/user-portal/server.js]: This is a temporally file. But I have added the license header till we keep it.

File: apps/user-portal/src/app.tsx (hunk 3 above)
Comment (jeradrutnam) [reply apps/user-portal/src/app.tsx]: Cleaned

File: apps/user-portal/src/locales/en.json (hunk 4 above)
Comment (jeradrutnam) [reply apps/user-portal/src/locales/en.json]: Fixed

**Comment by jeradrutnam** [comment]
> Better to use code formatting for the js files.

Yes, i'm already using, but IDE was forcing "no-newline" for json files. But anyhow now I got it fixed. 
And additionally code standard it tested by project code lint support.

**Pull request merged**